### ISSUE TITLE
"IsAlive"

### ISSUE THREAD

**Issue by steven1208** [Milor123/Motolora-Unlock#1]

**Comment by Milor123** [comment]
What do u mean? @steven1208

**Comment by steven1208** [comment]
![Screenshot_20220316-032224-915](https://user-images.githubusercontent.com/100505053/158569184-90ada861-d179-4855-b429-ef28b2947320.png)

**Comment by steven1208** [comment]
Still haven't been able to get it to work though

**Comment by Milor123** [comment]
Ohh understand, the reason is beacause is was developt with Pytho  2.7

**Comment by steven1208** [comment]
Do you think you could update this to be able to work properly? If so, that would be amazing!!!!

**Comment by Milor123** [comment]
Yes bro I could change it, but u stay sure about of my method, works it in the new devices? Do u read about of do it?

**Comment by steven1208** [comment]
I am unsure, I have tried it yes, but it won't let me get passed running the script because python-2.7 is no longer supported by win9 which I'm running win10. Let me know when you have the new version available and I'll give it a shot immidiatley, I have a a brand new Motorola g play prepaid from att&t that I have been trying to unlock to use but haven't had any luck, so even if it doesn't work or bricks the phone no harm but it's worth the shot
